src/include/sof/schedule/task.h

```diff
@@ -41,6 +42,11 @@ enum task_state {
 	SOF_TASK_STATE_RESCHEDULE,
 };
 
+struct task_ops {
+	enum task_state (*run)(void *data);
+	void (*complete)(void *data);
+};
```

Comment:
Can we doxygen all these members and structs.

Reply:
Done.